packages/tx/test/eip1559.spec.ts

```diff
@@ -15,6 +15,28 @@ const validSlot = Buffer.from('01'.repeat(32), 'hex')
 const chainId = new BN(4)
 
 tape('[FeeMarketEIP1559Transaction]', function (t) {
+  t.test('cannot input decimal values', (st) => {
+    const values = [
+      'maxFeePerGas',
+      'maxPriorityFeePerGas',
+      'chainId',
+      'nonce',
+      'gasLimit',
+      'value',
+      'v',
+      'r',
+      's',
+    ]
+    for (const value of values) {
+      const txData: any = {}
+      txData[value] = 10.1
```

Comment:
can we also try string decimal? `'10.1'`

Reply:
I have not checked anything related with strings - good point because this will probably also produce garbage output.

Reply:
awesome, trying to think of some other last cases we can add here: `-1`, `-1.1`, `'-1.1'`

Reply:
I'll add all cases which I also used in util :)